_locales/_experimental_en.json

```diff
@@ -2,6 +2,21 @@
   "login_instruction_desktop": {
     "message": "Login with an existing e-mail account (use your e-mail credentials here)"
   },
+  "keys_import": {
+    "message": "Import keys"
+  },
+  "keys_export": {
+    "message": "Export keys"
+  },
+  "keys_export_confirmation_message": {
+    "message": "Exporting your end-to-end setup! Keep these keys in a safe place and delete them as soon as they are not needed anymore."
+  },
+  "keys_export_success_message": {
+    "message": "Keys successfully exported to %1$s"
+  },
+  "settings_manage_keys": {
+    "message": "Manage Keys"
+  },
```

Comment:
Can't we use the already translated translation keys? I think they are adequate to the ones here. Maybe some leftovers of the old pr from @karissa? We now have access to all the translations which we also use on deltachat-android.
```
  "pref_managekeys_menu_title": {
    "message": "Manage keys"
  },
  "pref_managekeys_ask_for_storage_permission": {
    "message": "Delta Chat needs access to your files to import/export keys."
  },
  "pref_managekeys_export_secret_keys": {
    "message": "Export secret keys"
  },
  "pref_managekeys_export_explain": {
    "message": "Export secret keys to \"%1$s\"?"
  },
  "pref_managekeys_import_secret_keys": {
    "message": "Import secret keys"
  },
  "pref_managekeys_import_explain": {
    "message": "Import secret keys from \"%1$s\"?\n\n• Existing secret keys will not be deleted\n\n• The last imported key will be used as the new default key unless it has the word \"legacy\" in its filename"
  },
  "pref_managekeys_secret_keys_exported_to_x": {
    "message": "Secret keys written successfully to \"%1$s\"."
  },
  "pref_managekeys_secret_keys_imported_from_x": {
    "message": "Secret keys imported from \"%1$s\"."
  },
```